source/systems/gaea_user_guide.rst

```diff
@@ -91,2113 +93,1545 @@ points at Atlanta and Chicago.
+option to Slurm. Common dependency flags are summarized below. In each of these
+examples, only a single jobid is shown but you can specify multiple job IDs as
+a colon-delimited list (i.e. ``#SBATCH -d afterok:12345:12346:12346``). For the
+``after`` dependency, you can optionally specify a ``+time`` value for each
+jobid.
+
+.. table::
+    :widths: 25 75
+
+    +-----------------------------------+------------------------------------------------+
+    | Flag                              | Meaning (for the dependent job)                |
+    +===================================+================================================+
+    | ``#SBATCH -d after:jobid[+time]`` | The job can start after the specified jobs     |
+    |                                   | start or are canceled. The optional ``+time``  |
+    |                                   | argument is a number of minutes. If specified, |
+    |                                   | the job cannot start until that many minutes   |
+    |                                   | have passed since the listed jobs start/are    |
+    |                                   | canceled. If not specified, there is no delay. |
+    +-----------------------------------+------------------------------------------------+
+    | ``#SBATCH -d afterany:jobid``     | The job can start after the specified jobs     |
+    |                                   | have ended (regardless of exit state)          |
+    +-----------------------------------+------------------------------------------------+
+    | ``#SBATCH -d afternotok:jobid``   | The job can start after the specified jobs     |
+    |                                   | terminate in a failed (non-zero) state         |
+    +-----------------------------------+------------------------------------------------+
+    | ``#SBATCH -d afterok:jobid``      | The job can start after the specified jobs     |
+    |                                   | complete successfully (i.e. zero exit code)    |
+    +-----------------------------------+------------------------------------------------+
+    | ``#SBATCH -d singleton``          | Job can begin after any previously-launched    |
+    |                                   | job with the same name and from the same user  |
+    |                                   | have completed. In other words, serialize the  |
+    |                                   | running jobs based on username+jobname pairs.  |
+    +-----------------------------------+------------------------------------------------+
+
+
+Monitoring and modifying batch jobs
+-----------------------------------
+
+Holding and releasing jobs
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes you may need to place a hold on a job to keep it from starting. For
+example, you may have submitted it assuming some needed data was in place but
+later realized that data is not yet available. This can be done with the
```

Comment:
Replace "this can be done" with "You can do this"

Reply:
Updated in a new push.